packages/turf-point-grid/index.ts

```diff
@@ -12,13 +12,13 @@ import { distance } from "@turf/distance";
 import { point, featureCollection, Units } from "@turf/helpers";
 
 /**
- * Creates a {@link Point} grid from a bounding box, {@link FeatureCollection} or {@link Feature}.
+ * Creates a grid of points
  *
  * @name pointGrid
- * @param {Array<number>} bbox extent in [minX, minY, maxX, maxY] order
- * @param {number} cellSide the distance between points, in units
+ * @param {BBox} bbox extent of grid in [minX, minY, maxX, maxY] order
+ * @param {number} cellSide the distance between points
  * @param {Object} [options={}] Optional parameters
- * @param {string} [options.units='kilometers'] used in calculating cellSide, can be degrees, radians, miles, or kilometers
+ * @param {Units} [options.units='kilometers'] the units of the cellSide value.  Supports all valid Turf {@link https://github.com/Turfjs/turf/blob/master/packages/turf-helpers/README_UNITS.md Units}
```

Comment:
is this link going to work from the public docs site? (I think its going to send them to the github markdown page instead of staying on the docs website) Maybe that's fine for now, but it won't be versioned cleanly if we do it this way.

Reply:
When clicking the link from the public docs website, they will be sent to the markdown page, for now.  #2694 is meant to take it the rest of the way and extract the units into the turf-helpers README

Reply:
sounds good. happy to merge this as is and give you some breathing room to get that in a good place.